src/libraries/Native/Unix/configure.cmake

```diff
@@ -6,7 +6,7 @@ include(CheckPrototypeDefinition)
 include(CheckStructHasMember)
 include(CheckSymbolExists)
 include(CheckTypeSize)
-
+include(CheckFunctionExists)
```

Comment:
CheckFunctionExists is discouraged by CMake (https://cmake.org/cmake/help/latest/module/CheckFunctionExists.html), do you really need it?

Reply:
I can switch to `CheckSymbolExists`, however, the current supported platforms are detecting these two cases correctly.

Reply:
CheckSymbolExists is the recommended way be cmake so let's use it.

Reply:
Switched to `CheckSymbolExists`.
Btw, `CheckFunctionExists` is used in other places in this repo as well, which can be updated separately.